src/docs/implementations/dynatrace.adoc

```diff
@@ -4,25 +4,174 @@ Jon Schneider <jschneider@pivotal.io>
 :sectnums:
 :system: dynatrace
 
-Dynatrace is a dimensional time-series SaaS with built-in dashboarding.
+https://www.dynatrace.com/[*Dynatrace*] is a Software Intelligence Platform featuring application performance monitoring (APM), artificial intelligence for operations (AIOps), IT infrastructure monitoring, digital experience management (DEM), and digital business analytics capabilities.
+It can ingest multi-purpose dimensional time-series data and has built-in dashboarding. Both SaaS and self-hosted (Managed) deployments are offered.
 
 include::install.adoc[]
 
 == Configuring
 
+=== Setup
+
+For setting up new integrations with Dynatrace, it is recommended to use the latest version of the https://www.dynatrace.com/support/help/dynatrace-api/environment-api/metric-v2/[Dynatrace Metrics API] (`api-version: v2`).
+
+Use the following code in your project to export Micrometer metrics to the specified Dynatrace endpoint.
+If a Dynatrace OneAgent is installed on the host running Micrometer, metrics can be exported directly using the OneAgent without having to specify an endpoint URI or API token (see <<bookmark-oneagent, below>>).
+If no Dynatrace OneAgent is available, both the Dynatrace Metrics API v2 endpoint and an API token have to be specified.
+The https://www.dynatrace.com/support/help/dynatrace-api/basics/dynatrace-api-authentication/[Dynatrace API token documentation] contains more information on how to create an API token.
+The 'Ingest metrics' (`metrics.ingest`) permission is required on the token in order to ingest metrics.
+It is recommended to limit scope to only this permission.
+More information on how to use the v2 API is provided <<bookmark-apiv2, below>>.
+
 [source,java]
 ----
 DynatraceConfig dynatraceConfig = new DynatraceConfig() {
+    @Override
+    public DynatraceApiVersion apiVersion() {
+        return DynatraceApiVersion.V2;
+    }
+
+    @Override
+    public String uri() {
+        // The endpoint of the Dynatrace Metrics API v2 including path, e. g., "https://{your-environment-id}.live.dynatrace.com/api/v2/metrics/ingest".
```

Comment:
Please flip this to default to `""` and only put the endpoint URL placeholder into the comment.

Again - the default needs to be not having to provide the endpoint URL. Not having the OneAgent on the same host is the exception and not the rule.